cstore_fdw--1.5.sql

```diff
@@ -58,3 +60,37 @@ CREATE EVENT TRIGGER cstore_drop_event
     ON SQL_DROP
     EXECUTE PROCEDURE cstore_drop_trigger();
 
+CREATE TABLE public.pg_cstore_tables (logicalrelid regclass PRIMARY KEY);
+ALTER TABLE public.pg_cstore_tables SET SCHEMA pg_catalog;
+
+CREATE OR REPLACE FUNCTION public.register_cstore_table(table_name oid)
+	RETURNS boolean
+	LANGUAGE plpgsql STRICT
+	AS $rct$
+BEGIN
+    IF EXISTS(
```

Comment:
exposing a UDF for a simple select query looks like an overkill.

Moving implementation of register/unregister functions inside c code looks better to me.